packages/amplication-design-system/src/components/Loader/LoaderTiny.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Lottie, { LottieComponentProps } from "lottie-react";
+
+import animationData from "../../assets/amplication-loader-tiny.json";
+
+export type Props = Omit<LottieComponentProps, "animationData">;
+
+export const LoaderTiny = (props: Props) => {
```

Comment:
I think it makes more sense to use the current loader component and add a new type enum property instead of using a separate component

Reply:
I thought about that, but existing Loader component has some wrapper for full screen mode, I've decided not to do refactoring of this component and it's usage. More than that, not all of our loaders are going to be Lottie based. But if anyway we want to make a single Loader component that would accept animation type as parameter, let's do it separately from this issue. Should I open a new issue?